lib/machines.js

```diff
@@ -1164,6 +1253,21 @@ function create(req, res, next) {
         }
     }
 
+    if (req.params && req.params.volumes && req.params.volumes.length > 0) {
+        if (opts.brand === 'kvm') {
+            // At this time we do not support NFS volumes with KVM
+            return next(new InvalidArgumentError('volumes not yet supported ' +
```

Comment:
Should we add a test that checks that the relevant error is returned when trying to provision a KVM VM with volumes? That can be done in a subsequent PR, but we'd need to track that with a separate ticket.

Reply:
I'll add a test that does this. Sure.